imap_processing/cli.py

```diff
@@ -22,6 +23,8 @@
 
 import imap_processing
 
+__logger__ = logging.getLogger(__name__)
```

Comment:
Our pattern has been to set the local logger variable and then `logger.info()` later.
```suggestion
logger = logging.getLogger(__name__)
```